package.json

```
@@ -37,9 +37,8 @@
   "dependencies": {
     "underscore": "1.12.1",
     "command-node": "0.1.1",
-    "request": "2.88.0",
     "async": "1.5.2",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/jason-fox/iotagent-node-lib.git#feature/request-shim",
```

As described in https://github.com/telefonicaid/iotagent-node-lib/pull/1119#issuecomment-1000483835, this need to be addressed (reverted) as first step of the process. 

```suggestion
    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
```